src/libs/AxelarAssetTransfer.ts

```diff
@@ -53,19 +52,20 @@ export class AxelarAssetTransfer {
     fromChain: string,
     toChain: string,
     destinationAddress: string,
-    refundAddress?: string,
-    salt?: number
```

Comment:
i don't think we should delete the salt parameter. we can deafult to hashzero if the dApp doesn't want to pass it in, but allowing them to customize it (e.g. if they want to create their own nonces or something) is a feature we should keep

Reply:
Request from @milapsheth